layouts/partials/docs/legacy-warning.html

```diff
@@ -0,0 +1,10 @@
+<div class="pageinfo pageinfo-warning">
+    <h2>Legacy Flux is in Maintenance Mode</h2>
+    <p>You are looking at the documentation of Legacy Flux and Helm Operator (v1),
+       which is in maintenance mode.</p>
+    <p>The Flux community is working with most of its energy on its current inception
```

Comment:
Please remove this paragraph.

Reply:
What would you like to see in this version warning? Or which lines in particular would you like to see removed?

Reply:
I would simply remove the sentence because it can cause doubt about "production readiness".

> There is a new version -> upgrade now

Really is all that should be said.

Reply:
So remove lines 5-7? WFM

Reply:
```
Legacy Flux is in Maintenance Mode

This documentation is for Flux (v1) and Helm Operator (v1). Both projects are in maintenance mode and will soon reach end-of-life. We strongly recommend you familiarise yourself with the latest Flux release and start looking at your migration path.
```

Reply:
I would maybe rephrase it to something like:

```code
<h2>Upgrade Advisory</p>
<p>You are looking at the documentation of a version of Flux and the Helm Operator which are in maintenance mode, and have thereby become legacy projects.</p> 
<p>We strongly recommend you familiarise yourself with the newest Flux and
       <a href="/docs/migration/">start looking at your migration path</a>.</p>
```